pkg/lang/javascript/aws_runtime/aws.go

```diff
@@ -97,7 +99,7 @@ func (r *AwsRuntime) TransformPersist(file *core.SourceFile, annot *core.Annotat
 	importModule := ""
 	switch kind {
 	case core.PersistFileKind:
-		importModule = "fs"
+		importModule = "fs_" + annot.Capability.ID
```

Comment:
Does this need to be sanitized? I forget what (if any) restrictions we enforce on ids.

Reply:
so this is just for the import itself in the file. I cut a card so that we actually just pass the bucket name to the open call, but that requires a lot of extra work, so for now we are just importing an fs runtime per bucket and thats why we need different import modules